Chapters/6_Qualified_identifiers.md

```diff
@@ -25,6 +25,15 @@ designates the function `generate_intput_stream` that is found at lines 9 to 15
 Notice that the notion of "line number" is not always meaningful: the content
 may be a binary file, or a file that uses non standard line termination character(s).
 
+### 6.1.2 Bytes qualifier
+
+To overcome the limitations of the lines qualifier, the bytes qualifier allows
+to designate a byte range inside a content.  The range can be a single integer, or a pair of integers separated by `-`.
```

Comment:
The meaning of a `bytes=42` (without a byte *range*) is less intuitive than that of `lines=42`. It is clear that the latter still denotes a "range" (= the full content of a single line). While for the former there is potentially a doubt of whether it denotes 1 byte or just a position in a file (in "between" bytes).
I suggest stating explicitly in this section that a non-range bytes qualifier resolves to a 1-byte long "string".

Reply:
Also added suggeste  behaviour on non conformant SWHIDs.